src/main/java/com/example/flight/domains/Airport.java

```diff
@@ -0,0 +1,18 @@
+package com.example.flight.domains;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.NoArgsConstructor;
+
+@lombok.Data
+@NoArgsConstructor
+@ApiModel(description = "Domains to model the airports that are received form the FlightAPI")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Airport {
+    @ApiModelProperty(notes = "The code of the airport")
+    private String iata;
+    private String fs;
```

Comment:
What's the meaning of `fs` ??

Reply:
Is another code that the FlightAPI uses to identify an airport. Already put some documentation to it.